core-web/libs/dotcms-scss/angular/dotcms-theme/components/_progressbar.scss

```diff
@@ -3,7 +3,8 @@
     border: 0 none;
     height: 4px;
     background: $bg-highlight;
-    border-radius: $border-radius-xs;
+    border-radius: 0;
+    width: 4px;
```

Comment:
Where is this `width` coming from?

Reply:
Oh, that was me testing something, I forgot to delete it